### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by 0815pascal**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
-#### Q: How do i generate a token for the playground
-Solution
+#### Q: How do i generate a token for the playground?
+A: When working in the admin-view of the GraphQL-Playground (`http://localhost:4000/admin`), you are required to be authenticated. In order to do so, you have to create a session-token. You can do this in the following way:
```
Comment (roosnic1) [inline FAQ.md:2]: It's not the admin-view it's the graphQL API that needs authentication

File: FAQ.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+Copy the token and switch to the `HTTP Headers`-view. Now paste your token into the following code: 
+```
+{
+  "authorization": "Bearer rDFQbOKDLIG2JhDS6RveuIS6rRF7176R"
+}
+``` 
+As of now you’re good to go the execute queries/mutations in the GraphQL-Playground. 
+
+#### Q: What do I do if I receive the error `MongoError: Collection migrations already exists. Currently in strict mode` in the terminal?
+A: Basically this error means that you have a conflict in your database caused by installing two different (incompatible) branches.
```
Comment (roosnic1) [inline FAQ.md:35]: ...by working on two different branches with different migration states.

File: FAQ.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+{
+  "authorization": "Bearer rDFQbOKDLIG2JhDS6RveuIS6rRF7176R"
+}
+``` 
+As of now you’re good to go the execute queries/mutations in the GraphQL-Playground. 
+
+#### Q: What do I do if I receive the error `MongoError: Collection migrations already exists. Currently in strict mode` in the terminal?
+A: Basically this error means that you have a conflict in your database caused by installing two different (incompatible) branches. 
+
+You can solve this issue by running `docker-compose down` followed by `yarn clean`. These commands will delete the current database and remove any residuals.
```
Comment (roosnic1) [inline FAQ.md:37]: ... by stopping everything, running `docker-compose down` followed by `yarn clean && yarn build`.

File: FAQ.md [hunk 4]
```diff
@@ -1,2 +1,39 @@
...
+}
+``` 
+As of now you’re good to go the execute queries/mutations in the GraphQL-Playground. 
+
+#### Q: What do I do if I receive the error `MongoError: Collection migrations already exists. Currently in strict mode` in the terminal?
+A: Basically this error means that you have a conflict in your database caused by installing two different (incompatible) branches. 
+
+You can solve this issue by running `docker-compose down` followed by `yarn clean`. These commands will delete the current database and remove any residuals. 
+
+By then running `yarn dev` you can then setup the database from scratch and everything should be running again.
```
Comment (roosnic1) [inline FAQ.md:39]: Now if you start your database up and run `yarn watch` you'll get a fresh new database.

**Review by roosnic1 — CHANGES_REQUESTED** [review]

**Review by roosnic1 — APPROVED** [review]

**Review by faebebin — APPROVED** [review]
Good and understandable.

**Pull request merged**